integrationTests/chainSimulator/mempool/mempool_test.go

```diff
@@ -729,7 +736,7 @@ func Test_Selection_ShouldNotSelectSameTransactionsWithDifferentSenders(t *testi
 			PrevHash: []byte("blockHash0"),
 			RootHash: []byte(fmt.Sprintf("rootHash%d", 0)),
 		},
-		selectionSession,
+		accountsProvider,
```

Comment:
`OnProposedBlock()` now receives an accounts provider, instead of a selection session.